forest/src/cli/config.rs

```diff
@@ -31,6 +31,8 @@ pub struct Config {
     pub encrypt_keystore: bool,
     /// Metrics bind, e.g. 127.0.0.1:6116
     pub metrics_address: SocketAddr,
+    /// RPC bind, e.g. 127.0.0.1
```

Comment:
How about we use the same approach as in `metrics_address` and use `SocketAddr`? This way, on top of some uniformity, we'll have both the address and the port in one param.

Reply:
Good idea.

Reply:
Thanks. Yeah, it'd be great to specify the port purpose in the name.